packages/devtools_app/pubspec.yaml

```diff
@@ -62,12 +63,12 @@ dev_dependencies:
   devtools_testing: 0.9.8-dev.1
   flutter_test:
     sdk: flutter
+  freezed: ^0.12.0
```

Comment:
Fyi @kenzieschmoll for evaluation how much of an issue these extra package imports would cause in g3.